### ISSUE TITLE
Bump webdrivermanager to 5.4.x?

### ISSUE THREAD

**Issue by jungm** [primefaces/primefaces#10531]
We're experiencing issues with the current webdrivermanager (5.3.3) used by primefaces-selenium as it is not able to download the latest chromedriver versions:
`Aug 21, 2023 10:12:18 AM io.github.bonigarcia.wdm.versions.VersionDetector getDriverVersionFromRepository
WARNING: Exception reading https://chromedriver.storage.googleapis.com/LATEST_RELEASE_116 to get latest version of chromedriver (Error HTTP 404 executing https://chromedriver.storage.googleapis.com/LATEST_RELEASE_116)`

Leading to:
`org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: session not created: This version of ChromeDriver only supports Chrome version 114`

webdrivermanager [5.4.x](https://github.com/bonigarcia/webdrivermanager/blob/master/CHANGELOG.md#540---2023-06-21) fixes this as it uses the new chrome for testing endpoints to discover chromedriver builds

**BUT** webdrivermanager 5.4.x is built with java 11. Selenium is also dropping java 8 support in ~1 month: https://www.selenium.dev/blog/2023/java-8-support/

what do you think on this? This would at least require an IT environment to be on java 11+ while PF is still aiming for java 8 support and also running the integration-tests-23 job on java 8

**Comment by melloware** [comment]
They just released 5.5.0 supporting Java8 so I bumped.

**Comment by melloware** [comment]
Just a note I had to run clearDriverCache() and clearResolution() to fix my issues after upgrading to this new driver.

```java
       if (!System.getProperties().contains("webdriver.chrome.driver")) {
                webDriverManager = WebDriverManager.chromedriver();

                // uncomment if you need to clear cache getting Chrome mismatch errors
                //webDriverManager.clearDriverCache();
                //webDriverManager.clearResolutionCache();
            }
```

I only had to do this once.
